ovirt-engine.spec.in

```diff
@@ -1020,6 +1020,14 @@ OVESETUP_CONFIG/jbossHome=str:/opt/rh/eap7/root/usr/share/wildfly
 OVESETUP_CONFIG/pkiRenewalDocUrl=str:https://access.redhat.com/solutions/1572983
 __EOF__
 
+# Disable automatic keycloak integration in RHV.
+# Manual integration is possible and documented at ???
+# TODO: Provide a link. I found this, but think there are probably better ones:
```

Comment:
Please open an issue to track this

Reply:
Fixed the link and removed the TODO comment